src/server/database/models/MUser.ts

```diff
@@ -0,0 +1,9 @@
+import { sequelize } from "../database";
```

Comment:
lets import user model inside `database.ts` module passing `sequelize` instance into this module which will export function that will define User model